lib/testsetup.go

```diff
@@ -0,0 +1,77 @@
+package elastigo
```

Comment:
Ben asked me to put this file into its own package, and I tried to do so.  However, because circular package references are forbidden, I was not able to figure it out.  I tried to use an interface to access core elastigo functionality (see commented out below), but the method signatures for CreateIndex, etc. involve elastigo types.